### PR TITLE
updating readme

### PR DESCRIPTION
**Pull request by ciphereck**
Signed-off-by: Mudit Jain <ciphereck@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,16 +17,28 @@ When one gives laptop to his/her aquinatance, he/she can start locking session a
 
   
  ### Prerequisites  
-       
+  ```sh
+       1. Android Studio
```
Comment (akash1309) [inline README.md:21]: is it necessary to have android studio??

**Review by akash1309 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (akash1309) [reply README.md:21]: i think we can use any suitable editor like atom or any other.

Comment (ciphereck) [reply README.md:21]: @akash1309 android studio is not required necessarily. But generally, for android projects, we use an android studio, which in turn installs android and way to do Gradle build.
if one uses atom or another, he/she have to configure multiple things on their own i think.

Comment (akash1309) [reply README.md:21]: i think it can be done by titanium build.But lets build it on android , it will be easy.

**Review by akash1309 — APPROVED** [review]

**Pull request merged**